style.user.css

```
@@ -684,6 +684,7 @@ background-color: draculaW-blackSecondary
     --button-danger-bgColor-rest: var(--color-btn-bg);
     --button-danger-borderColor-rest: var(--color-btn-border);
     --counter-borderColor: var(--color-counter-border);
+    --bgColor-inset: var(--color-header-bg)
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--bgColor-inset is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)